transaction_pending.go

```diff
@@ -0,0 +1,40 @@
+package factom
+
+import (
+	"encoding/json"
+)
+
+// PendingTransaction is a single transaction returned by the pending-transaction API call
+type PendingTransaction struct {
+	TxID      string `json:"transactionid"`
+	DBHeight  uint32 `json:"dbheight"`
+	Status    string `json:"status"`
+	Fees      uint64 `json:"fees"`
+	Inputs    []TransactionAddress
+	Outputs   []TransactionAddress
+	ECOutputs []TransactionAddress
+}
+
+// GetPendingTransactions requests a list of transactions that have been
+// submitted to the Factom Network, but have not yet been included in a Factoid
+// Block.
+func GetPendingTransactions() ([]PendingTransaction, error) {
+	req := NewJSON2Request("pending-transactions", APICounter(), nil)
+	resp, err := factomdRequest(req)
+
+	if err != nil {
+		return nil, err
+	}
+	if resp.Error != nil {
+		return nil, err
```

Comment:
return resp.Error? Because err is nil at this point

Reply:
Good catch